active/operators/resynthesis/resynthesize_notes.py

```diff
@@ -1,9 +1,21 @@
-def resynthesize_notes_marked_as_deid(**kwargs):
+import json
+
+from datetime import datetime, timedelta
+from pymssql import OperationalError
+
+import utilities.common_variables as common_variables
+import utilities.common_hooks as common_hooks
+import utilities.common_functions as common_functions
+
+RESYNTH_JOB_ID = 'resynth_jobs'
```

Comment:
just want to call out - this is slightly different that we're using a common variable across scripts that's called from an operator...I trust your judgement, but maybe we can discuss at stand up tomorrow.